clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -266,7 +266,8 @@ public ApiException build(String message) {
             }
         }),
     INCONSISTENT_GROUP_PROTOCOL(23,
-            "The group member's supported protocols are incompatible with those of existing members.",
+            "The group member's supported protocols are incompatible with those of existing members" +
+                " or first group member tried to join with empty protocol.",
```

Comment:
There are two cases, right? The protocol type might be empty, or the list of protocols might be empty.